ECMAScript/react-native/example/App.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react'
+import { NavigationContainer } from '@react-navigation/native'
+import { createNativeStackNavigator } from '@react-navigation/native-stack'
+import { StatusBar } from 'expo-status-bar'
+import * as Linking from 'expo-linking'
+import { HomeScreen } from './screens/HomeScreen'
+import { ConnectorScreen } from './screens/ConnectorScreen'
+import { QuilttProvider } from '@quiltt/react'
+
+const Stack = createNativeStackNavigator()
+
+export default function App() {
+  const url = Linking.useURL()
+
+  useEffect(() => {
+    // log this url and set it to QuilttConnector.oauthRedirectUrl
+    console.log(url)
+  }, [url])
+
+  const token = 'PUT_A_REAL_TOKEN_HERE'
```

Comment:
We should update this to explain that this token should be coming from the customer's server.